internal/clients/client.go

```diff
@@ -33,7 +34,14 @@ func NewRESTConfig(kubeconfig []byte) (*rest.Config, error) {
 // NewKubeClient returns a kubernetes client given a secret with connection
 // information.
 func NewKubeClient(config *rest.Config) (client.Client, error) {
-	kc, err := client.New(config, client.Options{})
+	// in 0.15 this will be default
```

Comment:
Just merged a bump PR which updates controller runtime to [v0.15.1](https://github.com/crossplane-contrib/provider-kubernetes/blob/88915189d3837feb6ac826ef65a4061f843330a7/go.mod#L17), does this mean we no longer need the changes here?

Reply:
yes: https://github.com/kubernetes-sigs/controller-runtime/blob/40203bfeac472914f36ea35bf386326814f6f82e/pkg/client/client.go#L150